package.json

```diff
@@ -2,19 +2,23 @@
     "name": "react-prebid",
     "version": "2.1.1",
     "description": "Library for ad placements with Prebid header bidding in React applications",
-    "main": "index.js",
-    "esnext": "index-esnext.js",
+    "main": "lib/index.js",
```

Comment:
⚠️ when I run the `build` command, I get a lib folder that contains modules `Advertising` and `AdvertisingContext` as Common JS modules, but there is no `index.js` module – can you change the webpack config so that it creates a single index.js bundle?

Reply:
oops, ok fixed it now please try again.